File: samples/snippets/csharp/VS_Snippets_CLR/conceptual.formatting.overview/cs/overrides1.cs
namespace HotAndCold2
{
    // <Snippet2>
    public class Temperature
    {
        private decimal temp;

        public Temperature(decimal temperature)
        {
            this.temp = temperature;
        }

        public override string ToString()
        {
            return this.temp.ToString("N1") + "°C";
        }
    }

    public class Example12
    {
        public static void Main()
        {
            Temperature currentTemperature = new Temperature(23.6m);
            Console.WriteLine($"The current temperature is {currentTemperature}");
        }
    }
    // The example displays the following output:
    //       The current temperature is 23.6°C.
    // </Snippet2>
}

File: samples/snippets/visualbasic/VS_Snippets_CLR/conceptual.formatting.overview/vb/overrides1.vb
' Visual Basic .NET Document
Option Strict On

' <Snippet2>
Public Class Temperature
    Private temp As Decimal

    Public Sub New(temperature As Decimal)
        Me.temp = temperature
    End Sub

    Public Overrides Function ToString() As String
        Return Me.temp.ToString("N1") + "°C"
    End Function
End Class

Module Example13
    Public Sub Main13()
        Dim currentTemperature As New Temperature(23.6D)
        Console.WriteLine("The current temperature is " +
                          currentTemperature.ToString())
    End Sub
End Module
' The example displays the following output:
'       The current temperature is 23.6°C.
' </Snippet2>
